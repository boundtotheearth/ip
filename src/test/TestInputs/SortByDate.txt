event read book /at 2020-10-10
event read novel /at 2020-10-12
event read chapter /at 2020-10-11
sort date
