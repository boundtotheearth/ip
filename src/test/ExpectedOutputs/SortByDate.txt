Added: [E][✘] read book (at: 10 Oct 2020)
Added: [E][✘] read novel (at: 12 Oct 2020)
Added: [E][✘] read chapter (at: 11 Oct 2020)
Here are your tasks sorted by date
1. [E][✘] read novel (at: 12 Oct 2020)
2. [E][✘] read chapter (at: 11 Oct 2020)
3. [E][✘] read book (at: 10 Oct 2020)
